.travis.yml

```diff
@@ -1,52 +1,78 @@
-language: rust
-sudo: required
-dist: trusty
-addons:
-    apt:
-        packages:
-            - libssl-dev
-            - libcurl4-openssl-dev
-            - libelf-dev
-            - libdw-dev
-            - cmake
-            - gcc
-            - binutils-dev    
-cache: cargo
-rust:
-  - stable
-  - beta
-  - nightly
-env:
-  - RUSTFLAGS=-Ctarget-feature=+aes,+ssse3
-matrix:
-  allow_failures:
-    - rust: nightly
+rust: &rust
+  language: rust
+  sudo: required
+  dist: trusty
+  addons:
+      apt:
+          packages:
+              - libssl-dev
+              - libcurl4-openssl-dev
+              - libelf-dev
+              - libdw-dev
+              - cmake
+              - gcc
+              - binutils-dev    
+  cache: cargo
+  env:
+    - RUSTFLAGS=-Ctarget-feature=+aes,+ssse3
+  matrix:
+    allow_failures:
+      - rust: nightly
```

Comment:
Does this "allow failures" still work? When I look at: https://travis-ci.org/calvinaco/chain/builds/538754163

It doesn't have "nightly" separated out in "Allowed Failures" as in:
https://travis-ci.org/crypto-com/chain

Reply:
Oh yes, "allow failures" seems no longer effective. I will tweak the configuration and bring it back.

Reply:
It is backed now.